minio/api.py

```diff
@@ -1045,14 +1045,33 @@ def stat_object(self, bucket_name, object_name):
                                   object_name=object_name)
 
         etag = response.headers.get('etag', '').replace('"', '')
+        response.headers.pop('etag');
```

Comment:
One last comment, not sure what are you doing here? removing header ? for what reason?